packages/mobile/locales/es-419/global.json

```diff
@@ -106,7 +106,7 @@
   "qrFailedNoPhoneNumber": "El código QR no está asociado con el número de teléfono de tu contacto",
   "corruptedChainDeleted": "Se ha borrado información corrupta, por favor reinicie la applicación",
   "contractKitInitFailed": "No se pudo inicializar el kit de contrato, intente reinicializar la aplicación o enviar un correo electrónico de soporte.",
-  "web3FailedToSync": "Fallo la sincronización, por favor verifica tu conexión",
+  "web3FailedToSync": "Fallo la sincronización, por favor confirmado tu conexión",
```

Comment:
Same, `verifica` was fine here. Unrelated, but the first word (`Fallo`) should be `Falló`